cmd/common.go

```diff
@@ -358,11 +359,34 @@ func (repo *Repository) raisePullRequest() (string, error) {
 		Body  string `json:"body"`
 	}
 
+	pr_body := fmt.Sprintf(`
+	## What does this PR do?
+
+	This is an automated PR created by your security team to enable GitHub Code Scanning on your repository. This will allow us to find and fix security vulnerabilities in your code.
+
+	For more information on Code Scanning, please see [here](https://docs.github.com/en/code-security/code-scanning).
+
+	## How do I merge this PR?
+
+	This PR should have triggered a some CodeQL jobs to run. If these jobs have passed, you can merge this PR. If they have failed, please take a look at the logs to identify what went wrong and contact the security team if you require assistance.
```

Comment:
Small grammar issues - Consider rewording to `This PR should have triggered CodeQL scans for each [eligible](https://codeql.github.com/docs/codeql-overview/supported-languages-and-frameworks/) language in this repository`